samples/attested_tls/CMakeLists.txt

```
@@ -34,8 +34,8 @@ if ((NOT DEFINED ENV{OE_SIMULATION}) OR (NOT $ENV{OE_SIMULATION}))
       "Launch processes to establish an Attested TLS between an non-encalve TLS client and an TLS server running inside an enclave "
```

Pre-existing typo ...
```suggestion
      "Launch processes to establish an Attested TLS between an non-enclave TLS client and an TLS server running inside an enclave "
```